ros/lib/models.py

```diff
@@ -117,6 +118,7 @@ class RhAccount(db.Model):
     __tablename__ = 'rh_accounts'
     id = db.Column(db.Integer, primary_key=True)
     account = db.Column(db.Text, nullable=False)
```

Comment:
Please relax the `nullable=False` constraint from the `account` as further going we can have entries where account_id is not present.

Reply:
Something more good would be `check (account_id is not null or org_id is not null)`

Reply:
> Please relax the nullable=False constraint from the account as further going we can have entries where account_id is not present.

This is for Level-3 (handling anemic-tenants). As for now, account **must** be there.